src/repositories/adRepository.ts

```diff
@@ -103,7 +115,7 @@ export const get = async (
   sort?: any
 ): Promise<IAdModel[]> => {
   return (await Ad.find(filter || {})
-    .limit(limit || 20)
+    .limit(limit === null ? null : limit || 20)
```

Comment:
This line looks not clear for me. Strange ternary combined with ||. Please refactor. Possibly extract as a variable.